refinery/ui/source/js/dashboard/controllers/data-set-transfer-modal-ctrl.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Data Set Transfer Modal Ctrl
+ * @namespace DataSetTransferModalCtrl
+ * @desc Controller for modal to transfer a data set ownership to another user
+ * member to a group modal
+ * @memberOf refineryDashboard
+ */
+(function () {
+  'use strict';
+
+  angular
+    .module('refineryDashboard')
+    .controller('DataSetTransferModalCtrl', DataSetTransferModalCtrl);
+
+  DataSetTransferModalCtrl.$inject = [
+    '$log',
+    'dataSetV2Service',
+    'settings'
+  ];
+
+  function DataSetTransferModalCtrl (
+    $log,
+    dataSetV2Service,
+    settings
+  ) {
+    var vm = this;
+    vm.alertType = 'info';
+    vm.cancel = cancel;
+    vm.close = close;
+    vm.dataSetTitle = '';
+    vm.form = { email: '' };
+    vm.instanceName = settings.djangoApp.refineryInstanceName;
+    vm.isLoading = false;
+    vm.responseMessage = '';
+    vm.sendTransferRequest = sendTransferRequest;
+
+    /*
+     * ---------------------------------------------------------
+     * Methods Definitions
+     * ---------------------------------------------------------
+     */
+    /**
+     * @name cancel
+     * @desc  View method to cancel modals, expects modalInstance in scope
+     * @memberOf refineryApp.refineryDashboard.
+    **/
+    function cancel () {
+      vm.modalInstance.dismiss('cancel');
+    }
+
+    /**
+     * @name close
+     * @desc  View method to cancel modals, expects modalInstance in scope
+     * @memberOf refineryApp.refineryDashboard.
+    **/
+    function close () {
+      vm.modalInstance.close(vm.alertType);
+    }
+
+    /**
+     * @name generateAlertMessage
+     * @desc  helper method to generate api response notification
+     * @memberOf refineryApp.refineryDashboard.
+    **/
+    function generateAlertMessage (infoType, email, error) {
+      if (infoType === 'success') {
+        vm.alertType = 'success';
+        vm.responseMessage = 'Data set successfully transferred to ' + email;
+      } else if (infoType === 'danger') {
+        console.log(error);
```

Comment:
Do we keep console logging in the codebase?

Reply:
argh, $logger is preferred. Will remove.